packages/sanity/src/structure/panes/document/documentPanel/documentViews/useConditionalToast.ts

```diff
@@ -18,19 +10,20 @@ const LONG_ENOUGH_BUT_NOT_TOO_LONG = 1000 * 60 * 60 * 24 * 24
 export function useConditionalToast(params: ToastParams & {id: string; enabled?: boolean}) {
   const toast = useToast()
 
-  const wasEnabled = usePrevious(params.enabled)
```

Comment:
This is not necessary now that the cleanup function was added to the useEffect.
If the params.enabled condition was true, then the cleanup will take care of closing the toast when params.enabled changes or when the component is unmounted